k8s/values.yaml

```diff
@@ -7,14 +7,25 @@ applications:
   - name: celery
     image: pennlabs/penn-courses-backend
     secret: penn-courses
-    cmd: ["celery", "worker", "-A", "PennCourses", "-Q", "alerts,celery", "-linfo"]
+    cmd:
```

Comment:
this is not how yaml works. can't make newlines. Refer to below comment on how to newline this